draft/2022-08-17-this-week-in-rust.md

```diff
@@ -127,10 +105,12 @@ Rusty Events between 2022-08-17 - 2022-09-14 🦀
     * [**Rust for curious developers**](https://www.meetup.com/frontend_rm/events/287713743/)
 * 2022-08-20 | Virtual | [Rust Edu](https://rust-edu.org/workshop)
     * [**Rust Education Workshop 2022 (Submission deadline 2022-08-16)**](https://rust-edu.org/workshop)
-* 2022-08-20 | Virtual (Hoboken, NJ, US) | [Neighborhood Math Club](https://www.meetup.com/neighborhood-math-club/)
-    * [**Share Your Coding Project(s)!**](https://www.meetup.com/neighborhood-math-club/events/kbjcssydclbbc/)
+* 2022-08-24 | Virtual (Beijing, CN) | [WebAssembly and Rust Meetup (Rustlang)](https://www.meetup.com/wasm-rust-meetup/)
+    * [**[Tech Talk Live Appointment] Customize GitHub Workflow with Serverless Functions (How to use Rust and JavaScript to automate GitHub processes)**](https://www.meetup.com/wasm-rust-meetup/events/287876999/)
```

Comment:
No change needed here -- just an observation

This entire title (namely the inner-brackets and inner-parentheses) feel like a stress test on MD formatting 😹

Reply:
It definitely is exercising the formatting functionality -- I might fix up the title so that we are not doing that stress testing work here :)